ngraph/test/type_prop/unary_ops.cpp

```diff
@@ -96,6 +96,6 @@ REGISTER_TYPED_TEST_CASE_P(UnaryOperator,
                            dynamic_rank_input_shape_3D,
                            dynamic_rank_input_shape_full);
 
-using Types = ::testing::Types<op::Acos, op::Asin, op::Abs, op::Sqrt, op::Sin, op::Exp, op::Floor>;
+using Types = ::testing::Types<op::Acos, op::Asin, op::Abs, op::Sqrt, op::Sin, op::Exp, op::Floor, op::Log>;
```

Comment:
Yesterday I have discussed with @pszmel that will be good to split `unary_ops` tests as we already done for `arithmetic_ops`. @pszmel did you create a ticket for it?

Reply:
@ilyachur Not yet, but I will create it today.

Reply:
Ticket: CVS-57678. It is scheduled for next sprint.